lib/app/views/app_page.dart

```diff
@@ -197,11 +197,13 @@ class AppPage extends StatelessWidget {
             child: _buildTitle(context),
           ),
         builder(context, expanded),
-        if (actionsBuilder != null)
+        if (actionsBuilder != null &&
+            actionsBuilder!(context, expanded).isNotEmpty)
```

Comment:
Instead of invoking actionsBuilder twice, make a variable further up:
`final actions = actionsBuilder?.call(context, expanded) ?? [];`

Reply:
I agree! Pushed a commit that resolves this.